### ISSUE TITLE
aws provider iam_instance_profile_name not recognized

### ISSUE THREAD

**Issue by sshilko** [puphpet/puphpet#2114]
```
    # Let's use profiles
    aws.iam_instance_profile_name = "#{data['vm']['provider']['aws']['iam_profile_name']}"

```

needs to be added to Vagrantfile-aws

**Comment by jtreminio** [comment]
Can't find that:

https://github.com/mitchellh/vagrant-aws/search?utf8=%E2%9C%93&q=iam_profile_name

Did you mean `iam_instance_profile_name`?

**Comment by sshilko** [comment]
yes its important setting for AWS Security if u are relying on profiles, and just one line into Vagrant-aws

**Comment by jtreminio** [comment]
Can't use hiera directly because the values generated by the GUI aren't usually the exact `types`, like int or string or bool.

Many things also need massaged and verified in Puppet code before being passed back to modules.

I'll look into this `iam_instance_profile_name`

**Comment by sshilko** [comment]
gui can be whatever, but why limit manual editing hiera files. GUI doesnt even have to allow access to them.

i would say advanced users edit their hiera and manifests anyway, but well.. just proposal.
